l10n_it_withholding_tax/models/account.py

```diff
@@ -257,20 +257,21 @@ def default_get(self, fields):
                 rec["amount"] = invoice["amount_net_pay_residual"] * coeff_net
         return rec
 
-    def _compute_payment_amount(self, invoices=None, currency=None):
-        if not invoices:
-            invoices = self.invoice_ids
-        original_values = {}
-        for invoice in invoices:
-            if invoice.withholding_tax:
-                original_values[invoice] = invoice.residual_signed
-                invoice.residual_signed = invoice.amount_net_pay_residual
-        res = super(AccountAbstractPayment, self)._compute_payment_amount(
-            invoices, currency
-        )
-        for invoice in original_values:
-            invoice.residual_signed = original_values[invoice]
-        return res
+    # def _compute_payment_amount(self, invoices=None, currency=None):
```

Comment:
Secondo me meglio eliminare.

Reply:
Ciao @SirAionTech, ho aperto la PR solo per riportare una situazione che non mi è chiara, in seguito ad altri eventuali commenti (se qualcuno sa qualcosa a riguardo) pensavo di proporre una soluzione.

Reply:
Ho unito i commit, la funzione `_prepare_wt_values` è un residuo dalla v. 8.0, in cui c'era un wizard di creazione della dichiarazione RdA, e non è più utilizzata da allora.
La funzione `_compute_payment_amount` non esiste più, la sua funzionalità era di impostare il valore da pagare sulla fattura al valore residuo senza ritenuta, sostituita da altre funzioni e migliorata dalla funzione aggiunta in #4235

Reply:
Ho aperto la issue